api/package.json

```diff
@@ -144,6 +143,7 @@
 		"openid-client": "5.4.2",
 		"ora": "6.3.1",
 		"otplib": "12.0.1",
+		"papaparse": "^5.4.1",
```

Comment:
Let's pin the specific version, like the other dependencies

Reply:
Done